src/Attachment/EventumLegacyAdapter.php

```diff
@@ -33,11 +33,13 @@ private function getAttachment($path)
                     iaf_filetype,
                     iaf_filesize,
                     iaf_created_date,
-                    iaf_flysystem_path,
+                    iap_flysystem_path,
                     iaf_iat_id
-                FROM
-                    `issue_attachment_file`
-                WHERE
+                 FROM
+                    `issue_attachment_file`,
+                    `issue_attachment_file_path`
+                 WHERE
+                    iap_iaf_id = iaf_id AND
                     iaf_id=?';
```

Comment:
shouldn't delete also cleanup from `issue_attachment_file_path` table in legacy adapter?

Reply:
I don't think so.

All code is written requires a row in `issue_attachment_file_path` to exist for every row in `issue_attachment_file`. Deleting a file from the legacy adapter just sets `iaf_file` to be null so does not eliminate the need for `issue_attachment_file_path`.

If you are calling it from `Attachment::delete` then it could actually be a NOOP as the row is about to be deleted entirely, but it is logical to include it.

When calling `EventumLegacyAdapter::delete` from the migrate script (through the move method) I noticed I had just added in duplicate code which I have now removed in 73f8aa6